src/services/db/GitFileSystemService.ts

```diff
@@ -228,19 +228,17 @@ export default class GitFileSystemService {
   // Clone repository from upstream Git hosting provider
   clone(repoName: string): ResultAsync<string, GitFileSystemError> {
     const originUrl = `git@github.com:${ISOMER_GITHUB_ORG_NAME}/${repoName}.git`
-    const safeExistsSync = Result.fromThrowable(fs.existsSync, (error) => {
-      logger.error(`Error when checking if ${repoName} exists: ${error}`)
 
-      if (error instanceof GitError || error instanceof Error) {
-        return new GitFileSystemError("Unable to determine if directory exists")
-      }
-
-      return new GitFileSystemError("An unknown error occurred")
-    })
-
-    return safeExistsSync(`${EFS_VOL_PATH}/${repoName}`).asyncAndThen(
-      (isFolderExisting) => {
-        if (!isFolderExisting) {
```

Comment:
same here, do we not need these anymore?

Reply:
See above!